core/src/main/java/hudson/ClassicPluginStrategy.java

```diff
@@ -273,7 +273,7 @@ private void fix(Attributes atts, List<PluginWrapper.Dependency> optionalDepende
      * @see DetachedPluginsUtil#getImpliedDependencies(String, String)
      */
     @Deprecated // since TODO
-    @Nonnull
+    @CheckForNull
```

Comment:
Hm? https://github.com/jenkinsci/jenkins/blob/0f73231f775e06105103fa9d0f23e20d7b122ac1/core/src/main/java/jenkins/plugins/DetachedPluginsUtil.java#L83 is marked as nonnull

Reply:
oh sorry, my bad.

Reply:
@lacedx So what are the next steps here? It looks like this change is not valid